index.js

```diff
@@ -2900,6 +2947,14 @@
 
   return S;
 
+  };
+
+  //  Export two versions of the Sanctuary module: one with type checking;
+  //  one without.
+  var S       = createSanctuary(true);
+  S.unchecked = createSanctuary(false);
+  return S;
+
```

Comment:
> I'm reluctant to add complexity to the build step (which is currently non-existent). I'm not sure whether I'm being a pragmatic programmer (having learnt from experience) or whether I'm just being lazy. ;)

I don't know anything about publishing on npm, but I was just envisioning including actual files, eg

`alt/unchecked/index.js`
```js
module.exports = require('../../index.js').unchecked;
```
Does it not work like that?